client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/package-row.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { map, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ItemRow, { Header as ItemRowHeader } from './item-row';
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import Dropdown from 'woocommerce/woocommerce-services/components/dropdown';
+import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+import {
+	setContentsType,
+	setContentsExplanation,
+	setRestrictionType,
+	setRestrictionExplanation,
+	setAbandonOnNonDelivery,
+	setITN,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const PackageRow = ( props ) => {
+	const {
+		siteId,
+		orderId,
+		errors,
+		packageId,
+		translate,
+		contentsType,
+		contentsExplanation,
+		restrictionType,
+		restrictionExplanation,
+		abandonOnNonDelivery,
+		itn,
+		items,
+	} = props;
+	const abandonHandler = () => props.setAbandonOnNonDelivery( ! abandonOnNonDelivery );
+
+	return <div className="customs-step__package">
+		<label htmlFor={ packageId + '_abandonOnNonDelivery' }>
+			<Checkbox
+				id={ packageId + '_abandonOnNonDelivery' }
+				checked={ Boolean( abandonOnNonDelivery ) }
+				onChange={ abandonHandler } />
+			<span>{ translate( 'Abandon package if it can\'t be delivered' ) }</span>
+		</label>
+
+		<div className="customs-step__restrictions-row">
+			<div className="customs-step__contents-type">
+				<Dropdown
+					id={ packageId + '_contentsType' }
+					title={ translate( 'Contents type' ) }
+					value={ contentsType || 'merchandise' }
+					updateValue={ props.setContentsType }
+					valuesMap={ {
+						merchandise: translate( 'Merchandise' ),
+						documents: translate( 'Documents' ),
+						gift: translate( 'Gift' ),
+						sample: translate( 'Sample' ),
+						other: translate( 'Other…' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **71** times:
`translate( 'Other' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).